adapters/restclient/lib/opentelemetry/adapters/restclient/patches/request.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+# Copyright 2020 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module RestClient
+      module Patches
+        # Module to prepend to RestClient::Request for instrumentation
+        module Request
+          def execute(&block)
+            return super(&block) if disable_span_reporting?
+
+            trace_request do |span|
+              propagate_context(span)
+
+              super(&block)
+            end
+          end
+
+          # Override implementation (subclass) to determine per-connection
+          # span reporting rules.
+          def disable_span_reporting?
+            false
+          end
+
+          private
+
+          def trace_request
+            span = tracer.start_span(
+              url,
+              attributes: {
+                'component' => 'http',
+                'http.method' => method.to_sym,
```

Comment:
A symbol is not a valid attribute value. It should be a string.

Reply:
@mwear The Faraday adapter should be updated as well then. Should I handle that in this PR?

Reply:
Good catch (https://rubydoc.info/gems/faraday/Faraday/Env#method-instance_method). Go ahead and include it in this PR.